src/artemis/parameters/plan_specific/rotation_scan_internal_params.py

```diff
@@ -37,7 +37,10 @@ class RotationScanParams(BaseModel, AbstractExperimentParameterBase):
 
     @validator("rotation_direction", pre=True)
     def _parse_direction(cls, rotation_direction: str | int):
-        return RotationDirection[rotation_direction]
+        if isinstance(rotation_direction, str):
+            return RotationDirection[rotation_direction]
+        else:
+            return RotationDirection(rotation_direction)
```

Comment:
Nit: I would say we shouldn't allow the integer representation, only the string as it makes the API messier but not that fussed